src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -82,9 +82,15 @@ public async Task DeployCdkProject(OrchestratorSession session, CloudApplication
             var appSettingsFilePath = Path.Combine(cdkProjectPath, "appsettings.json");
 
             // Ensure region is bootstrapped
-            await _commandLineWrapper.Run($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion} -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
+            var cdkBootstrap = await _commandLineWrapper.TryRunWithResult($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion} -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
                 workingDirectory: cdkProjectPath,
-                needAwsCredentials: true);
+                needAwsCredentials: true,
+                redirectIO: true,
+                streamOutputToInteractiveService: true);
+
+            if (cdkBootstrap.ExitCode != 0)
+                throw new FailedToDeployCDKAppException(DeployToolErrorCode.FailedToRunCDKBootstrap, "We ran into an issue while running cdk bootstrap. This is the process of preparing your environment for deployment. Check the log/output for more details.");
```

Comment:
Pending verification on the message, otherwise LGTM.

Reply:
I looped Irene in to see if she has any issues

Reply:
I updated the message based on Irene's suggestion.